app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt

```diff
@@ -0,0 +1,135 @@
+package com.github.orkest.shazam.data
+
+import android.Manifest
+import android.media.AudioFormat
+import android.media.AudioRecord
+import android.media.MediaRecorder
+import android.util.Log
+import androidx.annotation.RequiresPermission
+import com.github.orkest.shazam.domain.ShazamConstants
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+
+class AudioRecording {
+
+    companion object {
+
+       @RequiresPermission(Manifest.permission.RECORD_AUDIO)
+       private fun buildAudioRecord(): AudioRecord {
+           //Set the audio source
+           val audioSource = MediaRecorder.AudioSource.UNPROCESSED
+
+           //Set the audio format
+           val audioFormat = AudioFormat.Builder()
+               .setChannelMask(AudioFormat.CHANNEL_IN_MONO)
+               .setEncoding(AudioFormat.ENCODING_PCM_16BIT)
+               .setSampleRate(48_000)
```

Comment:
Why is the audio sample rate at 48KHz? I know that it's also a standard alongside 44.1KHz but is there any specifc reason for this? A shazam requirement?

Reply:
Yes, it was just the proposed frequency by the Shazam tutorial documentation